components/inputs/input-date.js

```diff
@@ -445,6 +448,21 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 		this.focus();
 	}
 
+	async _open() {
+		if (this.disabled || this.skeleton) return;
```

Comment:
So what happens if it's in initially `<d2l-dropdown opened skeleton>` but then afterwards `skeleton` is removed -- will it still open? Similar for if it's initially opened and disabled.

Reply:
That's a good point - I would presume then in those cases we would want to open since opened would be set and that would be inconsistent if it wasn't actually open.

Reply:
Yeah that was my thinking as well.